ls1intum/Artemis
# [Feature] delete results

### Checklist

### Motivation and Context


Closes https://github.com/ls1intum/Artemis/issues/3412

### Description

Introduces a way to delete results of submissions. (Deletion of the whole submission is already possible)

### Steps for Testing


1. Navigate to Course Administration -> Exercise / ExerciseGroups -> Participations / Submissions
2. Now click on the number of Submissions of an participation /
![image](https://user-images.githubusercontent.com/33342534/119223316-fcbf8e00-baf8-11eb-9199-7434533b906a.png)
3. Next to every result there should be a delete button. 
![image](https://user-images.githubusercontent.com/33342534/119277919-99317f80-bc22-11eb-95b1-fc0901286631.png)
4. _Results_ should not be able to be deleted if a complaint is linked to them. Don't use the delete submissions button on the right, as it is not part of this PR. It will delete the submission no matter if it has results or complaints!

### Test Coverage



participation-submissions.component.ts: 
Assessment Resource:
Assessment Service:
